### ISSUE TITLE
[TBC][Raid][Karazhan][Boss] The Crone

### ISSUE THREAD

**Issue by countinbiskitsyo** [dalaranwow/dalaran-wow#3935]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Currently, she is not targettable after killing Dorothee and her three pets. Therefore you become "locked" on stage.
**Expected behaviour**: Tell us what should happen instead
She should enter a dialogue after killing Dorothee and her three pets and then engage in combat.
**Steps to reproduce the problem**:

1. Enter the "The Opera Hall" in Karazhan and defeat Dorothee and her 3 pets
2. The Crone spawns, but never goes into dialog and becomes untargettable and unattackable
3. you get locked on stage

**Include proofs for this behaviour**
![thecrone2](https://cloud.githubusercontent.com/assets/18533115/20914124/db3871d0-bb42-11e6-96b8-fc69f08e6de8.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**
![thecrone2](https://cloud.githubusercontent.com/assets/18533115/20914120/d1394ac4-bb42-11e6-833c-ece2d3ee49a9.png)

**Comment by Rushor** [comment]
Scripts/Karazhan: Update  the Opera event script to make Wizard of Oz actually completable.

Doing it in script instead of removing the flags from DB because I'm pretty sure that the DB flags are correct and there's some RP event missing before she becomes attackable.

* commit: 581792b07d168aee514167296f86086244ccf16b

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by jonathanargentiero** [comment]
Still reproducible today
